components/script/microtask.rs

```diff
@@ -38,6 +44,14 @@ pub struct EnqueuedPromiseCallback {
     pub pipeline: PipelineId,
 }
 
+/// A mutation callback
+#[derive(JSTraceable, HeapSizeOf)]
+pub struct EnqueuedMutationCallback {
+    #[ignore_heap_size_of = "Rc has unclear ownership"]
+    pub callback: Rc<MutationCallback>,
```

Comment:
Given the definition of https://dom.spec.whatwg.org/#notify-mutation-observers we should not need to store a callback here. I don't think we need a pipeline, either.